client/src/context/useDetailContext.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState, useContext, createContext, FunctionComponent } from 'react';
+import { cardDetailItems } from '../components/CardDialog/initialDetailData';
+import { IDialogItem, IDialogItemContext } from '../context/types/detail';
+
+const initialDetailData: IDialogItemContext = {
+  items: [] as IDialogItem[],
+  /* eslint-disable @typescript-eslint/no-unused-vars */
+  addItem: (_item: IDialogItem) => false,
+  removeItem: (_itemId: string) => false,
+  resetItems: () => false,
+};
+
+export const DialogContext = createContext<IDialogItemContext>(initialDetailData);
```

Comment:
I did this myself in my Kanban context work however, I recommend also changing this. Instead of the `initialDetailData` in the DialogContext intiializer you can just do `createContext<IDialogItemContext>({} as IDialogItemContext)`

Reply:
You mentioned this on our video call I think. Updated.